specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/cadl/multivariate/models.cadl

```diff
@@ -0,0 +1,370 @@
+  isAnomaly: boolean;
+
+  @doc("""
+Indicates the significance of the anomaly. The higher the severity, the more
+significant the anomaly is.
+""")
+  @minValue(0.0)
+  @maxValue(1.0)
+  severity: float32;
+
+  @doc("""
+Raw anomaly score of severity, will help indicate the degree of abnormality as
+well.
+""")
+  @minValue(0.0)
+  @maxValue(2.0)
+  score: float32;
+  interpretation?: AnomalyInterpretation[];
+}
+
+model AnomalyInterpretation {
+  @doc("Variable.")
+  variable?: string;
+
+  @doc("""
+This score shows the percentage contributing to the anomalous timestamp. A
+number between 0 and 1.
+""")
+  contributionScore?: float32;
+  correlationChanges?: CorrelationChanges;
+}
+
+model CorrelationChanges {
+  @doc("The correlated variables that have correlation changes under an anomaly.")
+  changedVariables?: string[];
+}
+
+@doc("""
+Training result of a model including its status, errors and diagnostics
+information.
+""")
+model ModelInfo {
+  @doc("""
+Source link to the input data to indicate an accessible Azure storage Uri,
+either pointed to an Azure blob storage folder, or pointed to a CSV file in
+Azure blob storage based on you data schema selection. 
+""")
+  dataSource: string;
+
+  @doc("""
+Data schema of input data source: OneTable or MultiTable. The default
+DataSchema is OneTable.
+""")
+  dataSchema?: DataSchema;
+
+  @doc("""
+A required field, indicating the start time of training data, which should be
+date-time of ISO 8601 format.
+""")
+  startTime: zonedDateTime;
+
+  @doc("""
+A required field, indicating the end time of training data, which should be
+date-time of ISO 8601 format.
+""")
+  endTime: zonedDateTime;
+
+  @doc("""
+An optional field. The display name of the model whose maximum length is 24
+characters.
+""")
+  @maxLength(24)
+  displayName?: string;
+
+  @doc("""
+An optional field, indicating how many previous timestamps will be used to
+detect whether the timestamp is anomaly or not. 
+""")
+  slidingWindow?: int32;
+
+  @doc("An optional field, indicating the manner to align multiple variables.")
+  alignPolicy?: AlignPolicy;
+
+  @doc("Model status. One of CREATED, RUNNING, READY, and FAILED.")
+  status?: ModelStatus;
+
+  @doc("Error messages when failed to create a model.")
+  errors?: ErrorResponse[];
+
+  @doc("Diagnostics information to help inspect the states of model or variable.")
+  diagnosticsInfo?: DiagnosticsInfo;
+}
+
+@doc("An optional field, indicating the manner to align multiple variables.")
+model AlignPolicy {
+  @doc("""
+An optional field, indicating how to align different variables to the same
+time-range. Either Inner or Outer. 
+""")
+  alignMode?: AlignMode;
+
+  @doc("""
+An optional field, indicating how missing values will be filled. One of
+Previous, Subsequent, Linear, Zero, Fixed.
+""")
+  fillNAMethod?: FillNAMethod;
+
+  @doc("An optional field. Required when fillNAMethod is Fixed.")
+  paddingValue?: float32;
+}
```

Comment:
Question to service, is it really every property is optional?

Reply:
@weidongxu-microsoft Yes, these are optional field, if the user doesn't specify those optional fields, we'll use the default setting.